boom-app/src/components/DisplayComponent.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import type { CsvOutput } from '@/types'
+
+defineProps<{ csvData: CsvOutput | undefined }>()
+</script>
+
+<template>
+  <h1>Observe that uploaded data!</h1>
```

Comment:
Je hebt in de UploadView al een `h1`. Dan is een `h2` hier logischer. Heading levels worden gebruikt om de structuur van de pagina duidelijk te maken aan bijvoorbeeld screenreaders. [Ter referentie](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks)